Code/C++/Union_of_two_unsorted_array.cpp

```diff
@@ -0,0 +1,87 @@
+// CPP program to find the Union of two unsorted arrays.
+
+/* Procedure 
+==================
+First of all here we'll initialize an empty hash set 's'.
```

Comment:
Hey just remove the word hash, as set is not a hash based data structure. Then your code is ready to merge!